scripts/shell_completions/bash/borg

```diff
@@ -18,7 +19,8 @@ _borg()
 	if [[ ${COMP_CWORD} == 1 ]] ; then
 		local borg_commands="init create extract check rename list diff delete prune info mount umount key serve upgrade recreate export-tar with-lock break-lock benchmark config"
 		COMPREPLY=( $(compgen -W "${borg_commands}" -- ${cur}) )
-		return 0
+		compopt +o default
+        	return 0
```

Comment:
Why is this indented?

Reply:
Ah I see, spaces are used instead of tabs. Now I don't know what the preferred method is in borg, but mixing tabs and spaces is not good.

Reply:
Sorry about the mixed indenting, using a new editor which made a little mess of the indenting. This one spilled by me. Do you want me to fix it in the request or will you adjust it when pulling? (I think it's just that one line).

Reply:
for python code we use 4 spaces per indentation level, no tabs.

and, if nothing speaks against that, I'ld prefer the same for other stuff, too.

Reply:
Yes, just checked it, and it would be nice if @balvik could convert the file please.

Reply:
I'll do it gladly after this PR is accepted, and will convert fish and zsh completions as well, so no worries.